internal/provider/common/series.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2024 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package common
```

Comment:
I see why you put this here (only used in MAAS), but I think it's better suited to being where it was in `core`.

Reply:
Sure thing